tck/src/test/java/org/reactivestreams/tck/PublisherVerificationTest.java

```diff
@@ -717,6 +735,79 @@ public void cancel() {
   }
 
   /**
+   * Verification using a Publisher that supports multiple subscribers
+   * @param shouldBlowUp if true {@link RuntimeException} will be thrown during second subscription.
+   */
+  final PublisherVerification<Integer> multiSubscribersPublisherVerification(final boolean shouldBlowUp) {
+    return new PublisherVerification<Integer>(newTestEnvironment()) {
+
+      @Override
+      public Publisher<Integer> createPublisher(final long elements) {
+        return new Publisher<Integer>() {
```

Comment:
Who tests the impls used in the tests of the tests... I honestly think that'd be overdoing it and not providing tangible value.

The tests (of tests) are by definition full of a-bit-wrong implementations designed to *specifically* trigger a *specific failure* and that's it. I don't see how it would help in maintenance or else to have them all be fully tested and compatible otherwise.